DOCS-15939: Remove ZStandard compression (#589)
mongodb/docs-node: source/fundamentals/connection/network-compression.txt

If you specify multiple compression algorithms, the driver selects the
first one in the list supported by your MongoDB instance.

.. _enable-compression:

Specify Compression Algorithms
------------------------------

You can enable compression for the connection to your MongoDB instance

> by specifying the algorithms using one of the following methods:

Reviewer: I: [Avoid "using" on its own](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#u)

S: 

```suggestion
by using one of the following methods to specify the algorithm:
```
Or reword the whole sentence:

```suggestion
To enable compression for the connection to your MongoDB instance,
use one of the following methods to specify the algorithm:
```

If you specify multiple compression algorithms, the driver selects the
first one in the list supported by your MongoDB instance.

.. _enable-compression:

Specify Compression Algorithms
------------------------------

> To enable compression for the connection to your MongoDB instance

Reviewer: ```suggestion
To enable compression for the connection to your MongoDB instance,
```

   .. tab:: MongoClientOptions
      :tabid: mongoclientoptions

>       To enable compression using `MongoClientOptions <{+api+}/api/global.html#MongoClientOptions>`__,

Reviewer: I: [Avoid "using" on its own](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#u)

S: 

```suggestion
      To enable compression with `MongoClientOptions <{+api+}/api/global.html#MongoClientOptions>`__,
```

Author: Thanks for identifying this. I think "by using" might be better. I avoid "with" because it usually signals the explanation isn't as specific as it could/should be. E.g.
"Connect to MongoDB with the driver" vs "Connect to MongoDB by using the appropriate method in the driver".
